### PR TITLE
feat: APA7 support

### LINKED ISSUE #9: Bibliography is using APA6 (supposed to be APA7)
The template uses apacite package which only supports APA6. The [bible](https://www.its.ac.id/pendidikan/wp-content/uploads/sites/112/2022/03/280-SK-Rektor-ttg-Pedoman-Penyusunan-Laporan-Tugas-Akhir-Sarjana-Sarjana-Terapan.pdf) says, 
>Format 3:2j :
>Semua sumber pustaka yang diacu secara langsung harus dicantumkan. Cara
mencantumkan sumber dapat dilihat pada sub bab 3.11. Standar penulisan sumber
pustaka yang disitasi di dalam bagian isi Laporan TA, dengan menggunakan standard
“APA” - Association Pshychology America edisi ke 7. ...

The weird bibliography in the template might have caused by the anger of the god for not adhering to the bible.

image

### PR DESCRIPTION
**Pull request by messier12**
breaking change: using biblatex, removed Indonesian bibliography.

bible in issue #9 seems to be using english for the bibliography anyway.

closes #9

### CODE REVIEW

**Review by LordRonz — APPROVED** [review]
lgtm

File: main.tex [hunk 1]
```diff
@@ -11,29 +11,27 @@
...
-\usepackage[utf8]{inputenc}
 
 % Pengaturan ukuran indentasi
 \setlength{\parindent}{2em}
 
 % Package lainnya
 \usepackage{changepage}
 \usepackage{etoolbox} % Mengubah fungsi default
-\usepackage{apacite} % APA-style citation
+%\usepackage{apacite} % APA-style citation
```
Comment (threeal) [inline main.tex:24]: please remove this line if no longer being used

File: main.tex [hunk 2]
```diff
@@ -71,6 +69,13 @@
 \counterwithin{figure}{section}
 \counterwithin{table}{section}
 
+% Mengganti figure dan table menjadi gambar dan tabel
+\renewcommand{\figurename}{Gambar}
+\renewcommand{\tablename}{Tabel}
+
+%\bibliographystyle{apacite}
```
Comment (threeal) [inline main.tex:76]: this one too

**Review by threeal — CHANGES_REQUESTED** [review]

File: main.tex (hunk 2 above)
Comment (messier12) [reply main.tex]: done

**Review by threeal — APPROVED** [review]

**Pull request merged**